tests/pytests/test_dawn_drivers.py

```diff
@@ -62,12 +20,14 @@ def test_kernels():
     for kern in binary_kernels:
         os.remove(kern)
 
-def test_dawn_load(test_kernels, usgscsm_compare_dict):
+def test_load(test_kernels):
     label_file = get_image_label('FC21A0038582_15170161546F6F')
-    usgscsm_isd = ale.load(label_file, props={'kernels': test_kernels}, formatter='usgscsm')
-    assert compare_dicts(usgscsm_isd, usgscsm_compare_dict) == []
-
+    compare_dict = get_isd("dawnfc")
 
+    isd_str = ale.loads(label_file, props={'kernels': test_kernels})
+    isd_obj = json.loads(isd_str)
+    print(json.dumps(isd_obj, indent=2))
```

Comment:
This only actually shows up if verbose is on right?

Reply:
It shows if the test fails.